hw_4-5.js

```diff
@@ -0,0 +1,320 @@
+"use strict"
+
+// NORMAL level
+
+// Task 1 
+
+function getSum(n) {
+
+    let sum = 0;
+
+    for (let i = 1; i <= n; i++) {
+        sum += i;
+    }
+    return sum;
+}
+
+console.log(getSum(100));
+
+// Task 2
+
+let rate = 0.17;
+let years = 5;
+
+let calcOverpay = amount => (amount * rate * years);
+
+console.log(calcOverpay(1000));
+
+// Task 3    ???
+
+function trimString(valueFrom, valueTo, str) {
+
+    for (let str = valueFrom; str <= valueTo; str++) {
+        console.log(str);
+    }
+    return str;
+}
+
+trimString(5, 10);
+
+// Task 4
+
+function getSumNumbers(num) {
+
+    let value = 0;
+    let numToStr = String(num);
+
+    for (let i = 0; i < numToStr.length; i++) {
+        value += +(numToStr[i]);
```

Comment:
так норм, но скобки лишние
```suggestion
        value += +numToStr[i];
```